Basic/DistributedAuthoritySocialHub/Assets/Editor/Tutorials/Tutorial0_SetupUGS/3-CheckVivox.asset

```diff
@@ -0,0 +1,166 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: ff771ccdf4150419d9ff4d342b069aae, type: 3}
+  m_Name: 3-CheckVivox
+  m_EditorClassIdentifier: 
+  Title:
+    m_Untranslated: Connect the Vivox Voice Chat service
+  m_Paragraphs:
+    m_Items:
+    - m_Type: 6
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: 
+      m_CodeSample: 
+      m_PostInstructionImage: {fileID: 0}
+      m_Tutorial: {fileID: 0}
+      m_Image: {fileID: 0}
+      m_VideoUrl: 
+      m_Video: {fileID: 0}
+      m_CriteriaCompletion: 0
+      m_Criteria:
+        m_Items: []
+      m_MaskingSettings:
+        m_MaskingEnabled: 1
+        m_UnmaskedViews:
+        - m_SelectorType: 1
+          m_ViewType:
+            m_TypeName: 
+          m_FocusEditorWindow: 1
+          m_EditorWindowType:
+            m_TypeName: UnityEditor.ProjectSettingsWindow, UnityEditor.CoreModule,
+              Version=0.0.0.0, Culture=neutral, PublicKeyToken=null
+          m_AlternateEditorWindowTypes:
+            m_Items: []
+          m_MaskType: 0
+          m_MaskSizeModifier: 0
+          m_UnmaskedControls: []
+      m_Summary: 
+      m_Description: 
+      m_InstructionBoxTitle: 
+      m_InstructionText: 
+      m_TutorialButtonText: 
+    - m_Type: 0
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: "To initialize Vivox Voice Chat service:\n\n1. Go to <b>Edit</b>
+          > <b>Project Settings</b>. \n\n2. Select the <b>Services</b> tab. \n\n3.
+          Select the <b>Vivox</b> tab.\n\nThis automatically adds your Unity Cloud
+          credentials and enables Vivox Services on your project."
```

Comment:
As this is done automatically when visiting the page, it's weird that this is an instructive page, because no action from the user is required. It should instead be a narrative page that tells the user that the system automagically set up things for them 2 seconds ago.

Reply:
The thing is, I still think this is valuable to point out, even if IETs automatically opens the Vivox tab.
You have to do this step, or else the server address field inside the Vivox section never becomes populated, and leads to some initialization issues the user might encounter.

Reply:
I'm not saying to not point this out. I'm saying to replace an **Instructive** page with a **Narrative** one (IET has pages of different types, and Narrative ones don't require you to complete "tasks").
As users don't have to do anything, we could have a Narratieve page that simply says something along the line of "We just set up this for you, and if you want to check it you can open this section of the project settings from <path>"

Reply:
@fernando-cortez after the latest commits the page no longer works: it no longer opens the **Services > Vivox** tab for you, and therefore Vivox is not initialized. After changing the page, did you re-add the code/callbacks that automated the setup?

Reply:
Seems fixed now!